### DISCUSSION TITLE
Display Information after Submitted Form

### DISCUSSION THREAD

**Discussion by Jasminahmad** [quasarframework/quasar#13719] [General - Components / Directives / etc]
I have a form that after filled in and submit, I want the data to be displayed on the main page. Is there any way on how to display data of a form after submitting without using any backend API? Just dummy data only. Doesn't need to be stored in any database.

**Comment by pratik227** [comment]
@Jasminahmad  You can use Local Storage or Session Storage to store the data.

https://quasar.dev/quasar-plugins/web-storage#introduction

Or 

https://quasar.dev/quasar-cli-vite/state-management-with-vuex#introduction

**Reply by alexrififi** [reply]
Perhaps Pinia is a better choice.

https://quasar.dev/quasar-cli-vite/state-management-with-pinia

**Reply by pratik227** [reply]
Yes vuex is depricated by vue js team. You can use pinia
